app/components/blacklight/document/page_header_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render blacklight_config.track_search_session.applied_params_component.new if blacklight_config.track_search_session.applied_params_component %>
```

Comment:
Can this be moved into a method on the component? Something like `<%= applied_params %>`